spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -52,6 +52,32 @@ class LanguageServerPlugin(SpyderCompletionPlugin):
     TIME_BETWEEN_RESTARTS = 10000  # ms
     TIME_HEARTBEAT = 3000  # ms
 
+    # --- Signals
+    # ------------------------------------------------------------------------
+    sig_exception_occurred = Signal(dict)
+    """
+    This Signal is emitted to report that an exception has occured.
+
+    Parameters
+    ----------
+    error_data: dict
+        The dictionary containing error data. The expected keys are:
+        >>> error_data = {
+            "text": str,
+            "is_traceback": bool,
+            "repo": str,
```

Comment:
I think `repo` doesn't apply in this case because this error comes from an internal plugin.

Reply:
Fixed